sklearn/cluster/optics_.py

```diff
@@ -221,8 +222,8 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : array, shape (n_samples, n_features), or (n_samples, n_samples)  \
-if metric=’precomputed’.
+        X : array, shape (n_samples, n_features), or (n_samples, n_samples)
+            if metric=’precomputed’.
```

Comment:
Better not to make any unrelated changes. This change breaks the rendering of `OPTICS.fit`:
[current PR](https://65862-843222-gh.circle-artifacts.com/0/doc/modules/generated/sklearn.cluster.OPTICS.html) vs [master](https://scikit-learn.org/dev/modules/generated/sklearn.cluster.OPTICS.html)